debian/control

```diff
@@ -15,6 +15,7 @@ Build-Depends: bash-completion,
                python3-lxml,
                python3-magic,
                python3-petname,
+               python3-pexpect,
```

Comment:
we don't run the integration tests on package build just in adt, reason for this not being needed